### ISSUE TITLE
Phase of the Moon

### ISSUE THREAD

**Issue by paviad** [MarScaper/ephemeris#9]
Question - Is there a way using this library to determine the phase of the moon at a given time point?

Thanks!

**Comment by Lebostein** [comment]
Yes! I have the same question. I use for other projects in Python the library "PyEphem" to calculate sunrise, sunset and moon phase: https://rhodesmill.org/pyephem/

Calculating the moon phase (float between 0 and 1) there is very simple:

_import ephem
moon = ephem.Moon()
observer = ephem.Observer()
observer.lat  = '52.520008'
observer.long = '13.404954'
observer.date = ephem.now()
moon.compute(observer)
print(moon.moon_phase)_

Now for an ESP32 project I took a look at this library, which promises a similar range of functions. Sunrise and sunset work. But I haven't yet found out how to calculate the moon phase...
